<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGraphics</class>
 <widget class="QWidget" name="ConfigureGraphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>509</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleName">
   <string>Graphics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="apiBox">
     <property name="title">
      <string>API Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="apiLayout"/>
      </item>
      <item>
       <widget class="QWidget" name="deviceWidget" native="true">
        <layout class="QVBoxLayout" name="deviceLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="rendererBox">
     <property name="title">
      <string>Renderer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="advancedBox">
     <property name="title">
      <string>Advanced</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>270</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
